code-challenge-android-kotlin/app/build.gradle

```diff
@@ -36,6 +36,12 @@ dependencies {
     implementation "io.reactivex.rxjava2:rxandroid:2.0.2"
     implementation 'com.github.bumptech.glide:glide:4.6.1'
 
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutines_version"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$coroutines_version"
+    implementation 'ru.gildor.coroutines:kotlin-coroutines-retrofit:0.9.0'
+
+    implementation "android.arch.lifecycle:extensions:$lifecycle_version"
```

Comment:
Why did you choose to use Architecture Components?

Reply:
Because the task "Remove logic from the HomeActivity" required to remove the logic from the views. I thought it was a good idea to use LiveData and ViewModel since the views could observe changes in the viewmodel, avoiding memory leaks after the view is destroyed. Second, to be able to keep the data after orientation changes for the other task. Furthermore, I intented to follow this guide https://developer.android.com/jetpack/docs/guide, implementing the repository layer too, if there was some time left at the end of the tasks.

Reply:
Thanks for your clarification, Alice!